pkg/rehearse/jobs.go

```diff
@@ -303,7 +360,18 @@ func (jc *JobConfigurer) ConfigurePeriodicRehearsals(periodics config.Periodics)
 	filteredPeriodics := filterPeriodics(periodics, jc.loggers.Job)
 	for _, job := range filteredPeriodics {
 		jobLogger := jc.loggers.Job.WithField("target-job", job.Name)
-		if err := jc.configureJobSpec(job.Spec, jc.loggers.Debug.WithField("name", job.Name)); err != nil {
+		variant := ""
+		if newVariant, ok := job.Labels[jobconfig.ProwJobLabelVariant]; ok {
```

Comment:
nit: this is not really a new variant, maybe `variantLabel` or something could be a more appropriate name